db/seeds.rb

```diff
@@ -5,3 +5,108 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+
+# Default admin
+
+User.create(
+  email: "admin@subway-reviews.com",
+  password: "password",
+  password_confirmation: "password",
+  role: "admin"
+  )
+
+lines = Line.create([
+  {name: "Red"},
+  {name: "Green"},
+  {name: "Blue"},
+  {name: "Orange"},
+  ])
+
+require 'open-uri'
+require 'nokogiri'
+require 'json'
+
+green_line_stops =[]
+red_line_stops =[]
+orange_line_stops =[]
+blue_line_stops =[]
+silver_line_stops =[]
```

Comment:
Useless assignment to variable - `silver_line_stops`.<br>Surrounding space missing for operator '='.